gocov/test.go

```diff
@@ -21,12 +21,113 @@
 package main
 
 import (
+	"bytes"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
 )
 
+func capture(wd string, args []string) ([]byte, error) {
```

Comment:
This function should be renamed, or the coverage-specific bits moved elsewhere. Probably the latter.